lib/vdsm/storage/lvm.py

```diff
@@ -279,10 +279,16 @@ def _buildFilter(devices):
 
 
 def _buildConfig(dev_filter, use_lvmpolld="1"):
+    if dev_filter:
+        lvm_filter = "filter={}".format(dev_filter)
+    else:
+        lvm_filter = ""
```

Comment:
I think we can simplify this to:

    def _build_config(dev_filter="", use_lvmpolld="1"):
        if dev_filter:
            dev_filter =  f"filter={dev_filter}"